packages/editor/src/components/block-compare/index.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import classname from 'classnames';
```

Comment:
Just a typo here, the imported function should be `classnames` instead of `classname`